ai2thor/tests/test_unity.py

```diff
@@ -203,10 +201,10 @@ def test_fast_emit(controller):
 
     assert event.metadata["actionReturn"] is None
     assert event_fast_emit.metadata["actionReturn"] == "foo"
-    assert id(event.metadata["objects"]) == id(event_fast_emit.metadata["objects"])
-    assert id(event.metadata["objects"]) != id(event_no_fast_emit.metadata["objects"])
-    assert id(event_no_fast_emit_2.metadata["objects"]) != id(
-        event_no_fast_emit.metadata["objects"]
+    assert id(event._metadata["objects"]) == id(event_fast_emit._metadata["objects"])
+    assert id(event._metadata["objects"]) != id(event_no_fast_emit._metadata["objects"])
+    assert id(event_no_fast_emit_2._metadata["objects"]) != id(
+        event_no_fast_emit._metadata["objects"]
```

Comment:
This change was made because `.metadata` now creates a new object from `._metadata` after it is called for the first time from an event. So, the `id` of the newly created `.metadata` object of course changes.